src/services/v2/adapters/vcs-adapter.js

```diff
@@ -0,0 +1,27 @@
+export const VcsAdapter = {
```

Comment:
Since the service is for API v4, maybe use v4 instead of v2 on the path?
src/services/`v2`/adapters/vcs-adapter.js